src/GraphQL.FluentValidation/ValidatorTypeCache.cs

```diff
@@ -9,14 +9,39 @@
 namespace GraphQL.FluentValidation
 {
     /// <summary>
-    /// Static cache for all <see cref="IValidator"/>.
```

Comment:
This cache never was static :)

Reply:
It's a "per-execution-level" kinda cache isn't it. I wonder the comment could say something like that.